scripts/send_log_config.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+from argparse import RawTextHelpFormatter
+import argparse
+import json
+import socket
+import struct
+
+
+parser = argparse.ArgumentParser(
+        description="""
+Send a python log config (json) to a port where a logging process listens.
+
+Futher deatils about the log configuration format and usage can be found here:
+https://docs.python.org/2/library/logging.config.html """,
+        formatter_class=RawTextHelpFormatter)
+
+parser.add_argument('-c', action="store", required="True",
+                    dest="config_file",
+                    help="Log configuration in json format.")
+
+parser.add_argument('-p', action="store", required="True",
+                    dest="port", type=int,
+                    help="Port number of the logger server.")
+
+args = parser.parse_args()
+
+try:
+    with open(args.config_file, 'rb') as cf:
+        log_config = cf.read()
+
+        # Just a simple check for valid json before sending
+        json.loads(log_config)
+
+        data_to_send = log_config
+
+    host = 'localhost'
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+    print('connecting ...')
```

Comment:
It would be better to print the host and the port: `print('Connecting on {0} port {1}...'.format(host, args.port))`